compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp

```diff
@@ -729,6 +729,14 @@ struct QuantizeActivation final : public luci::CircleNodeMutableVisitor<bool>
         circle_node->quantparam()->scale.push_back(scaling_factor);
         circle_node->quantparam()->zerop.push_back(zp);
       }
+      // Fix special attributes
```

Comment:
@jinevening 
I have a question about design of this Visitor:
Why do we quantize node inputs instead of node itself?